spec/models/customer_spec.rb

```diff
@@ -32,4 +32,78 @@
       expect(build(:customer)).to be_valid
     end
   end
+
+  describe 'open quotes' do
+    it 'returns only open quotes' do
+      customer = create(:customer)
+      company = create(:company)
+      customer_request1 = create(:customer_request,
+        customer_id: customer.id
+        )
+      customer_request2 = create(:customer_request,
+        customer_id: customer.id
+        )
+      quote1 = create(:quote,
+        customer_request_id: customer_request1.id,
+        company_id: company.id
+      )
+      create(:quote,
+        customer_request_id: customer_request2.id,
+        company_id: company.id,
+        accepted: true
+      )
+      expect(customer.open_quotes).to eq([quote1])
+    end
+
+    it 'should not return accepted quotes' do
+      customer = create(:customer)
+      company = create(:company)
+      customer_request2 = create(:customer_request,
+        customer_id: customer.id
+        )
+      quote2 = create(:quote,
+        customer_request_id: customer_request2.id,
+        company_id: company.id,
+        accepted: true
+      )
+      expect(customer.open_quotes).not_to include([quote2])
+    end
+  end
+
+  describe 'accepted_quotes' do
```

Comment:
I think generally there would be some better ways to set these tests up. We aren't creating a situation that could go wrong by not creating multiple quotes, some accepted and some not.  Lets go over these next time.